tests/dtm/test_single_process_restart.py

```diff
@@ -347,3 +359,145 @@ def test_bkt_creation_ios_after_m0d_restart(self):
                       "sizes objects.")
 
         self.log.info("ENDED: Verify bucket creation and IOs after m0d restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-41225")
+    def test_continuous_read_during_m0d_restart(self):
+        """Verify continuous READ during m0d restart using pkill."""
+        self.log.info("STARTED: Verify continuous READ during m0d restart using pkill")
+        bucket_name = 'bucket-test-41225'
+        object_prefix = 'object-test-41225'
+        log_file_prefix = 'test-41225'
+        que = multiprocessing.Queue()
+
+        self.log.info("Step 1: Start write Operations :")
+        self.dtm_obj.perform_write_op(bucket_prefix=bucket_name,
+                                      object_prefix=object_prefix,
+                                      no_of_clients=self.test_cfg['clients'],
+                                      no_of_samples=self.test_cfg['samples'],
+                                      obj_size=self.test_cfg['size'],
+                                      log_file_prefix=log_file_prefix, queue=que)
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+        workload_info = resp[1]
+        self.log.info("Step 2: Start READ Operations :")
+        proc_read_op = multiprocessing.Process(target=self.dtm_obj.perform_ops,
+                                               args=(workload_info, que,
+                                                     True,
+                                                     True,
+                                                     True, 10))
+        proc_read_op.start()
+
+        self.log.info("Step 3 : Perform Single m0d Process Restart During Read Operations")
+        resp = self.dtm_obj.process_restart(master_node=self.master_node_list[0],
+                                            health_obj=self.health_obj,
+                                            pod_prefix=POD_NAME_PREFIX,
+                                            container_prefix=MOTR_CONTAINER_PREFIX,
+                                            process=self.m0d_process, check_proc_state=True)
+        assert_utils.assert_true(resp, "Failure in observed during process restart/recovery")
```

Comment:
"in" not needed here, check wherever applicable

Reply:
fixed